src/Logging/Logging.Console/src/AnsiLogConsole.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.Extensions.Logging.Console
     /// </summary>
     internal class AnsiLogConsole : IConsole
     {
+        private const string _defaultForegroundColor = "\x1B[39m\x1B[22m";
```

Comment:
```suggestion
        private const string DefaultForegroundColor = "\x1B[39m\x1B[22m";
```
I'm not 100% sure about naming guidelines for consts in this repo, but I assume it should be PascalCase.

Reply:
👍